src/www/diag_ipsec_spd.php

```diff
@@ -73,9 +73,9 @@
                 <tr>
                   <td colspan="2"></td>
                   <td colspan="3">
-                    <span class="fa fa-arrow-right fa-fw" aria-hidden="true" alt="in"></span>
+                    <span class="fa fa-arrow-right fa-fw" aria-hidden="true"></span>
                     <?= gettext("incoming (as seen by firewall)"); ?> <br/>
-                    <span class="fa fa-arrow-left fa-fw" aria-hidden="true" alt="out"></span>
+                    <span class="fa fa-arrow-left fa-fw" aria-hidden="true"></span>
```

Comment:
Probably title was meant -> creates a tooltip

Reply:
I know `title` creates a tooltip, but I didn't think one was intended - it's all in lower case.
Of course, I could add one now - but what shall I take as text? And shall I add gettext as well?